terraform/deployments/mobile-backend/gha-iam-role.tf

```diff
@@ -0,0 +1,50 @@
+locals {
+  github_oidc_arn = "arn:aws:iam::${data.aws_caller_identity.current.account_id}:oidc-provider/token.actions.githubusercontent.com"
+}
+
+data "aws_iam_openid_connect_provider" "github_oidc" {
+  url = "https://token.actions.githubusercontent.com"
+}
+
+data "aws_iam_policy_document" "config_signing_role_permissions" {
+  statement {
+    actions = [
+      "kms:DescribeKey",
+      "kms:GetPublicKey",
+      "kms:Sign"
+    ]
+    resources = [aws_kms_key.config_signing_key.arn]
+  }
+}
+
+data "aws_iam_policy_document" "gha_image_attestation_trust" {
```

Comment:
I think most of the resource names in this file still have references to "gha_image"

Reply:
sorry yes good spot have changed all of those now